_assets/javascripts/components/countdown.js

```diff
@@ -122,6 +122,7 @@ CRDS.Countdown = class Countdown {
     Countdown.setLoadingStatus(true);
     return CRDS.Countdown.getEvents()
       .done((events) => {
+        this.events = events;
```

Comment:
what is the point of this line? Does it get used ever?

Reply:
It gets pretty heavily used in the next 5/6 lines of code I think.
@lukemoderwell

Reply:
Jimmy clarified, needed.